python/samples/documentation_examples/plugins/MathPlugin/math.py

```diff
@@ -0,0 +1,103 @@
+import math
+import sys
+
+if sys.version_info >= (3, 9):
+    from typing import Annotated
+else:
+    from typing_extensions import Annotated
+
+from semantic_kernel.functions.kernel_function_decorator import kernel_function
+
+
+class Math:
+    """
+    Description: MathPlugin provides a set of functions to make Math calculations.
+
+    Usage:
+        kernel.import_plugin_from_object(MathPlugin(), plugin_name="math")
+
+    Examples:
+        {{math.Add}} => Returns the sum of input and amount (provided in the KernelArguments)
+        {{math.Subtract}} => Returns the difference of input and amount (provided in the KernelArguments)
+        {{math.Multiply}} => Returns the multiplication of input and number2 (provided in the KernelArguments)
+        {{math.Divide}} => Returns the division of input and number2 (provided in the KernelArguments)
+    """
+
+    @kernel_function(
+        description="Divide two numbers.",
+        name="Divide",
+    )
+    def divide(
+        self,
+        input: Annotated[int, "the first number to divide from"],
```

Comment:
inconsistent input, this one is int, others are str and float...

Reply:
I don't follow: number2 is string? Follow dotnet's example, everything should be a float.

Reply:
We should also make the arguments match dotnet: number1 and number2.